common/common-ui/src/main/java/com/duckduckgo/common/ui/view/TypeAnimationTextView.kt

```diff
@@ -57,7 +57,7 @@ class TypeAnimationTextView @JvmOverloads constructor(
                 }
             }
         }
-
+        if (typingAnimationJob?.isActive == true) typingAnimationJob?.cancel()
         typingAnimationJob = launch {
```

Comment:
`typingAnimationJob` should be a `ConflatedJob` so that we don't need to do this.

```kotlin
private var typingAnimationJob = ConflatedJob()
```

```suggestion
        if (typingAnimationJob?.isActive == true) typingAnimationJob?.cancel()
        typingAnimationJob += launch {
```